tools/docker/README.md

```diff
@@ -35,9 +35,9 @@ The docker runner image can be used to run prplMesh inside containers using `run
 
 The runner docker allows running multiple containers, for example one with a controller+agent, the other with an agent only:
 The 2 containers are connected using a docker network (bridge), so can
-communicate + sniffed by running `wireshark` / `tcpdump` on the bridge from the host to see 1905 packets.
+communicate + sniffed by running `wireshark` / `tshark` / `tcpdump` on the bridge from the host to see 1905 packets.
 This is what is done by the test scripts in the tests/ directory.
-Therefore they require tcpdump and tshark to be installed in order to work.
+Therefore they require dumpcap and tshark to be installed in order to work.
```

Comment:
dumpcap is part of the `wireshark-common` package, so perhaps it should be mentioned here.

Reply:
tshark package is sufficient because it depends on wireshark-common.